Prac05/countWords.py

```diff
@@ -0,0 +1,24 @@
+string = input("Enter a string: ")
+list_of_words = string.lower().split()
+
+word_dict = {}
+
+for word in list_of_words:
+    if word in word_dict:
+        word_dict[word] += 1
+    else:
+        word_dict[word] = 1
+
+word_lengths = []
+words = sorted(list(word_dict.keys()))
+for word in words:
+    word_lengths.append(len(word))
+
+width = max(word_lengths)
```

Comment:
Try using a one-line list comprehension (generator) to calculate the maximum length.